app/graphql/types/repository_type.rb

```diff
@@ -54,131 +92,13 @@ class RepositoryType < BaseObject
           "The number of downloads according to the Counter Code of Practice."
   field :citation_count,
         Integer,
-        null: true, description: "The number of citations."
-
-  field :datasets,
```

Comment:
I'm not sure, why you need to delete these ones for the repository type. I know the repository page in commons won't use it. But why would you remove it from the API completely?

Reply:
i guess the same comment goes for software, publications, dmps, etc.

Reply:
Reverted